lib/Minz/HookType.php

```diff
@@ -0,0 +1,66 @@
+<?php
+declare(strict_types=1);
+
+enum Minz_HookType: string {
+	case ApiMisc = 'api_misc';	// function(): void
+	case BeforeLoginBtn = 'before_login_btn';	// function(): string
+	case CheckUrlBeforeAdd = 'check_url_before_add';	// function($url) -> Url | null
```

Comment:
```suggestion
	case CheckUrlBeforeAdd = 'check_url_before_add';	// function(string $url): string|null
```

Favour PHP native typing syntax when possible

Reply:
done